rebuild_wrappers.py

```diff
@@ -620,7 +630,7 @@ def add_symbol_name(funname: Union[str, None], funsname: Dict[ClausesStr, List[s
 		print("(Also, the program WILL crash later if you proceed.)")
 		sys.exit(2) # Check what you did, not proceeding
 	
-	return gbl, redirects, filespec
+	return gbl, redirects, filespec, alltypedefs
```

Comment:
You forgot to update the return signature.
(Also, the file should be accepted by `mypy` if you comment the imports you don't use at the beginning of the file, and this error should have been caught by this tool.)